.changeset/healthy-radios-notice.md

```diff
@@ -0,0 +1,7 @@
+---
+'@sigstore/mock': patch
+---
+
+Bump dependencies:
+
+- nock from 13.3.4 to `13.3.6
```

Comment:
been meaning to fix my action so it parses the version properly :) 

kinda annoying how dependabot says oclif/core was updated but only in lockfile.. so ideally we would only get a changeset when the version requirement is updated in `package.json`?